webapp/src/Service/EventLogService.php

```diff
@@ -54,6 +55,17 @@ class EventLogService implements ContainerAwareInterface
     // TODO: Add a way to specify when to use external ID using some (DB)
     // config instead of hardcoding it here. Also relates to
     // AbstractRestController::getIdField.
+    /**
```

Comment:
Stuff like this is a little bit annoying

Reply:
I could drop this, but then we need a small change in the logic below, since it assumed stuff that wasn’t true

Reply:
I will see what it takes to not do this since it’s indeed a bit verbose

Reply:
I changed this to another way of doing it, this seems shorter/better/less of a hassle.